src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -624,7 +624,7 @@ public function tempnam($dir, $prefix)
      * @param string $filename The file to be written to
      * @param string $content  The data to write into the file
```

Comment:
I agree, updated on the two newly added methods. 
Since this targets master, would you mind to open a PR against 2.7 for changing this docblock? I think it's worth it